api/src/main/resources/org/openmrs/api/db/hibernate/Condition.hbm.xml

```diff
@@ -21,12 +21,14 @@
 				<param name="sequence">condition_condition_id_seq</param>
 			</generator>
 		</id>
-
 		<many-to-one name="previousVersion" class="org.openmrs.Condition" column="previous_version" />
 		<property name="onsetDate" type="java.util.Date"
 				  column="onset_date" length="19"  />
-		<property name="condition" type="java.lang.String"
-				  not-null="true" length="50" column="condition" />
+		<component name="condition" class="org.openmrs.CodedOrFreeText">
```

Comment:
@paradisekelechi did you know about how to you hibernate components before? Or have you just read about it now?

Reply:
I just read it up so as to do the task

Reply:
You are a fast learner! 👍